server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java

```diff
@@ -306,25 +311,25 @@ public TabletServerResourceManager(ServerContext context, TabletHostingServer ts
         Property.TSERV_MINC_MAXCONCURRENT, true);
     modifyThreadPoolSizesAtRuntime(
         () -> context.getConfiguration().getCount(Property.TSERV_MINC_MAXCONCURRENT),
-        "minor compactor", minorCompactionThreadPool);
+        METRICS_TSERVER_MINOR_COMPACTOR_POOL, minorCompactionThreadPool);
 
-    splitThreadPool = ThreadPools.getServerThreadPools().getPoolBuilder("splitter")
-        .numCoreThreads(0).numMaxThreads(1).withTimeOut(1, SECONDS)
-        .enableThreadPoolMetrics(enableMetrics).build();
+    splitThreadPool = ThreadPools.getServerThreadPools()
+        .getPoolBuilder(METRICS_POOL_PREFIX + "tserver.minor.compactor").numCoreThreads(0)
```

Comment:
I think this is misnamed, should indicate split in the name and not minor compaction.

Reply:
Will change to `METRICS_POOL_PREFIX + "tserver.split"`

Reply:
Fixed in 21aa29c435

Reply:
I think putting `tserver` in the name is not a good choice. When merged to elasticity this will need to change as splits are done in the manager.